utils/prepare/hashes.py

```diff
@@ -0,0 +1,73 @@
+import argparse
```

Comment:
So many new lines of code... Isn't it possible to reuse some utils from Linux?
What about https://www.tensorflow.org/api_docs/python/tf/keras/utils/get_file?
Think about maintenance efforts!